src/llmcompressor/modifiers/obcq/base.py

```diff
@@ -83,25 +83,12 @@ class SparseGPTModifier(Modifier):
     prunem_: Optional[int] = None
     compressible_layers_: Optional[List] = None
 
-    def on_initialize_structure(self, state: State, **kwargs):
-        """
-        Initialize the structure of the model for compression.
-        This modifier does not modifiy the model structure, so this method
-        is a no-op.
-
-        :param state: session state storing input model and calibration data
-        """
-        return True
-
     def on_initialize(self, state: "State", **kwargs) -> bool:
         """
         Initialize and run the OBCQ algorithm on the current state
 
         :param state: session state storing input model and calibration data
         """
-        if not self.initialized_structure_:
-            self.on_initialize_structure(state, **kwargs)
```

Comment:
why is this check/condition ok to remove?

Reply:
Because `on_initialize_structure` does not do anything. Nearly all `on_initialize_structure` calls outside of GPTQ are no-ops, and GPTQ's `on_initialize_structure` should be depreciated anyways